ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -16,7 +16,15 @@ ${ACTIONS_BTN_XP}=    xpath=//div/button[@aria-label="Actions"]
 ${DELETE_ACTION_BTN_XP}=  xpath=//div/ul/li/button[text()="Delete project"]
 ${EDIT_ACTION_BTN_XP}=    xpath=//div/ul/li/button[text()="Edit project"]
 ${SPAWNER_LINK}=        xpath=//*[text()="Launch Jupyter"]
-
+${NB_IMAGE}=        Minimal Python
+${NB_IMAGE_GPU}=        PyTorch
+${PV_NAME_TOL_1}=         ods-ci-tolerations
+${PV_NAME_TOL_2}=         ods-ci-tolerations-2
+${PV_NAME_GPU}=         ods-ci-gpu
+${PV_DESCRIPTION}=         it is a PV created to test DS Projects feature
+${PV_SIZE}=         1
```

Comment:
please bring these back to the test suite file

Reply:
Done